### DISCUSSION TITLE
Add filters on workflows like frontpages filters

### DISCUSSION THREAD

**Discussion by Giorgos-Hellene** [YetiForceCompany/YetiForceCRM#14352] [Feature improvement]
Hi company !! 

I have to suggest a small feature. Please take a look.

I believe that would be better if there was the same filtes that there are at Accounts, Quote, Invoice etc to workflows.

Some times i try to apply some taks but they are not possible or it demands more time to achieve my goal. 
If there was this feature we can do it easier with less time.

Here are the filters from frontpages 
![εικόνα](https://user-images.githubusercontent.com/50912403/110317621-129ed600-8015-11eb-872f-279887d6ced6.png)

and here are the filters from workflows
![εικόνα](https://user-images.githubusercontent.com/50912403/110317684-28ac9680-8015-11eb-9526-33480ccb7210.png)

What do you think? Will it be convenient for us ?
 
Furthermore if there was the filters of frontpages on workflow home page it will be better for all administrator or manager.
![εικόνα](https://user-images.githubusercontent.com/50912403/110318062-af617380-8015-11eb-938b-f4c3511798b7.png)

Thank you

**Comment by mariuszkrzaczkowski** [comment]
I agree with you, we thought about that many times. The problem is that it's a huge and time consuming change. The entire workflow mechanism needs to be refactored, it's a leftover from Vtiger that is very hard to make, and nobody is willing to pay for it. There are so many great mechanisms that could be combined with workflows, which would make the functionality even more expanded. 

This is a very old code and logic that is not compliant with new rules/trends of writing code and needs to be written from scratch.
